webapp/portlet/src/main/webapp/vue-apps/vertical-menu/VerticalMenuService.js

```diff
@@ -0,0 +1,16 @@
+export function getSite(siteType, siteName, lang) {
+  const formData = new FormData();
+  formData.append('lang', lang);
+  formData.append('siteName', siteName);
+  const params = new URLSearchParams(formData).toString();
+  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/sites/details/${siteType}?${params}`, {
```

Comment:
IMO you can calculate the current site id like [here](https://github.com/exoplatform/layout-management/blob/develop/layout-management-webapps/src/main/webapp/WEB-INF/jsp/siteNavigation.jsp#L14-L16) and use the existing getSiteById rest service